tests/test_main.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
 import unittest
 
-from linked_list import Node, LinkedList
+from linked_list.node import Node
+from linked_list.implementation import LinkedList
```

Comment:
Nop, actually, we want `Node` and `LinkedList` to be part of the public global interface of the `linked_list` package. If we do `from linked_list.implementation import LinkedList` we're "seeing" the implementation details and we want to be abstracted away from it.